pkg/logconfigurations/types.go

```diff
@@ -0,0 +1,92 @@
+	LogConfigurations LogConfigurationsAccountStitchedFields `json:"logConfigurations,omitempty"`
+	//
+	Name string `json:"name,omitempty"`
+}
+
+// Actor - The `Actor` object contains fields that are scoped to the API user's access level.
+type Actor struct {
+	// The `account` field is the entry point into data that is scoped to a single account.
+	Account Account `json:"account,omitempty"`
+}
+
+// LogConfigurationsAccountStitchedFields -
+type LogConfigurationsAccountStitchedFields struct {
+	// Look up for all obfuscation expressions for a given account
+	ObfuscationExpressions []LogConfigurationsObfuscationExpression `json:"obfuscationExpressions"`
+}
+
+// LogConfigurationsCreateObfuscationExpressionInput - Input for creating an obfuscation expression.
+type LogConfigurationsCreateObfuscationExpressionInput struct {
+	// Description of expression.
+	Description string `json:"description,omitempty"`
+	// Name of expression.
+	Name string `json:"name"`
+	// Regex of expression.
+	Regex string `json:"regex"`
+}
+
+// LogConfigurationsObfuscationExpression - Reusable obfuscation expression.
+type LogConfigurationsObfuscationExpression struct {
+	// Identifies the date and time when the expression was created.
+	CreatedAt DateTime `json:"createdAt"`
+	// Identifies the user who has created the expression.
+	CreatedBy UserReference `json:"createdBy,omitempty"`
+	// Description of the expression.
+	Description string `json:"description,omitempty"`
+	// The id of the obfuscation expression.
+	ID string `json:"id"`
+	// Name of the expression.
+	Name string `json:"name"`
+	// Regular expression for this obfuscation expression. Capture groups will be obscured on matching.
+	Regex string `json:"regex"`
+	// Identifies the date and time when the expression was last updated.
+	UpdatedAt DateTime `json:"updatedAt"`
+	// Identifies the user who has last updated the expression.
+	UpdatedBy UserReference `json:"updatedBy,omitempty"`
+}
+
+// LogConfigurationsUpdateObfuscationExpressionInput - Input for updating an obfuscation expression.
+// Null fields are left untouched by mutation.
+type LogConfigurationsUpdateObfuscationExpressionInput struct {
+	// Description of expression.
+	Description string `json:"description,omitempty"`
+	// Expression Id.
+	ID string `json:"id"`
+	// Name of expression.
+	Name string `json:"name,omitempty"`
+	// Regex of expression.
+	Regex string `json:"regex,omitempty"`
+}
+
+// UserReference - The `UserReference` object provides basic identifying information about the user.
+type UserReference struct {
+	//
+	Email string `json:"email,omitempty"`
+	//
+	Gravatar string `json:"gravatar,omitempty"`
+	//
+	ID int `json:"id,omitempty"`
+	//
+	Name string `json:"name,omitempty"`
+}
+
+type obfuscationExpressionsResponse struct {
+	Actor Actor `json:"actor"`
+}
+
+// DateTime - The `DateTime` scalar represents a date and time. The `DateTime` appears as an ISO8601 formatted string.
+type DateTime string
```

Comment:
We usually override the `DateTime` scalar with `nrtime.DateTime`. Take a look at this example in the `tutone.yml` https://github.com/newrelic/newrelic-client-go/blob/main/.tutone.yml#L366

Reply:
done